internal/agents/jiraretrieval.go

```
@@ -0,0 +1,234 @@
+	defer r.Body.Close()
+	webReq, err := jira.TransformJiraWebhook(body)
+	if err != nil {
+		http.Error(w, "Invalid webhook payload", http.StatusBadRequest)
+		return
+	}
+	if err := j.ProcessWebhook(r.Context(), webReq); err != nil {
+		http.Error(w, fmt.Sprintf("Failed to process webhook: %v", err), http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte(fmt.Sprintf("Webhook processed for ticket %s", webReq.TicketID)))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)